superset-frontend/src/explore/components/DatasourcePanel/DatasourcePanel.test.tsx

```diff
@@ -174,8 +174,69 @@ test('should render a warning', async () => {
         },
       },
     }),
+    { useRedux: true },
   );
   expect(
     await screen.findByRole('img', { name: 'alert-solid' }),
   ).toBeInTheDocument();
 });
+
+test('should render a create dataset infobox', () => {
+  render(
+    setup({
+      ...props,
+      datasource: {
+        ...datasource,
+        type: DatasourceType.Query,
+      },
+    }),
+    { useRedux: true },
+  );
+
+  screen.logTestingPlaygroundURL();
```

Comment:
delete?

Reply:
Whoops! Good catch, I missed that one. Removed in [`this commit`](https://github.com/apache/superset/pull/19855/commits/5d7be1141a9c33ce1b5af9365c074cbe167fb4eb).